logdna/resource_view_test.go

```diff
@@ -292,8 +292,8 @@ func TestViewJSONUpdateError(t *testing.T) {
 	levels2 := "critical"
 	host1 := "host1"
 	host2 := "host2"
-	category1 := "DEMO"
-	category2 := "DEMO2"
+	category1 := "DemoCategory1"
+	category2 := "DemoCategory2"
```

Comment:
minor, but given that we're testing against state that we cannot control under test (that is, categories that must already exist), it might be good to put back assertion(s) that demonstrate that the category name stored in state matches the _server_ and not what was provided.

for example, send `DEMOCATEGORY2` and assert that we get back the appropriate case

Reply:
The categories on the server are mixed case, but appear upper case in the UI.  Are you suggesting to make those upper case, or just provide upper for the test and assert the mixed case?

Reply:
Done.